examples/send_dates_transaction.js

```diff
@@ -0,0 +1,45 @@
+const pkg = require('../package.json');
+const { Exporter } = require('../index')
+const exporter = new Exporter(pkg.name, true)
+
+async function pushData() {
+  const timestamp = Math.floor(new Date() / 1000)
+
+  let lastPosition = await exporter.getLastPosition()
+  console.log(`Last position: ${JSON.stringify(lastPosition)}`)
+  let key = 1
+  if(lastPosition) {
+    key = lastPosition.key + 1
+  }
```

Comment:
You can make use of the ternary operator to avoid double assignments:
```js
let key = lastPosition ? lastPosition.key +1 : 1
```

Reply:
Yes, much more readable.